assets/elm/main.elm

```diff
@@ -328,6 +415,31 @@ nameTagView model =
         ]
 
 
+playersListView : Model -> Html msg
+playersListView model =
+    div []
+        [ h2 [] [ text "Painters" ]
+        , ul [] <| displayPlayer model.players
+        ]
+
+
+displayPlayer : List Player -> List (Html.Html msg)
+displayPlayer players =
+    List.map
+        (\player ->
+            li []
+                [ text ("Name: " ++ player.name)
+                , ul
+                    []
+                    [ li [] [ text ("Player Id: " ++ (player.player_id |> toString)) ]
+                    , li [] [ text ("Seat: " ++ (player.seat |> toString)) ]
+                    , li [] [ text ("Active Player? " ++ (player.isActive |> toString)) ]
```

Comment:
why all the `++ (x |> toString)`s? How about just `++ toString x`

Reply:
Ya that's better!